crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1661,7 +1667,7 @@ impl TransportRpcModules {
     /// Returns [Ok(false)] if no http transport is configured.
     pub fn merge_http(&mut self, other: impl Into<Methods>) -> Result<bool, RegisterMethodError> {
         if let Some(ref mut http) = self.http {
-            return http.merge(other.into()).map(|_| true)
+            return http.merge(other.into()).map(|_| true);
```

Comment:
```suggestion
            return http.merge(other.into()).map(|_| true)
```

Reply:
Got rugpulled by rustfmt being trigger-happy.
Undone in 3e5f678f8431096f36ec7185de038b7a72b2b0c4